docs/package.json

```diff
@@ -67,5 +69,17 @@
     "setupFiles": [
       "<rootDir>/loadershim.js"
     ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
Can we talk about husky as a team before we merge?

Reply:
Happy to discuss, perhaps it's overkill.